src/pug/index.pug

```diff
@@ -7,6 +7,11 @@ mixin site-content
   .columns.is-centered
     .column.is-half.has-text-centered
 
+mixin hajime-panel
```

Comment:
Are we using this panel anywhere?

(I'm just leaving these comments because I want to see netlify working)

Reply:
Changes made